functions/global.sh

```diff
@@ -14,6 +14,7 @@ source /app/libexec/post_update.sh
 source /app/libexec/prepare_component.sh
 source /app/libexec/presets.sh
 source /app/libexec/configurator_functions.sh
+source /app/libexec/cloud_sync.sh
```

Comment:
We should consider if it's possible to source this only if cloud_sync is actually enabled in the config, and when enabled this should be sourced even there as if we insert this check we are not sure if it was sourced at the startup.